hedera-node/hedera-app/src/main/java/com/hedera/node/app/services/OrderedServiceMigrator.java

```diff
@@ -68,12 +68,11 @@ public List<StateChanges.Builder> doMigrations(
             @Nullable final SoftwareVersion previousVersion,
             @NonNull final SoftwareVersion currentVersion,
             @NonNull final Configuration config,
-            @NonNull final NetworkInfo networkInfo,
+            @Nullable final NetworkInfo genesisNetworkInfo,
```

Comment:
Isn't the ordered service migrator called in non-genesis cases as well? 

Hedera.java:
```
    * Initialization Step 2: Initialize the state. Either genesis or restart or reconnect or some other trigger.
    * Includes migration when needed.
```

Reply:
True. But the networkInfo is used only in genesis case. So I wanted to clarify and renamed it.